src/Calendar.js

```diff
@@ -649,6 +649,10 @@ class Calendar extends React.Component {
       eventContainerWrapper: elementType,
       dayWrapper: elementType,
       dateCellWrapper: elementType,
+      /**
+       * component used as a header for each column in the TimeGridHeader
+       */
+      header: elementType,
```

Comment:
actually this is already documented below in the specific views segment of this proptype

Reply:
I'm not sure I follow. What's holding this PR back?

Reply:
The PR isn't needed the header component is already documented here